src/by/epam/ayem/module5/model/Treasure.java

```diff
@@ -33,4 +47,10 @@ public double getTreasureCost() {
     public void setTreasureCost(double treasureCost) {
         this.treasureCost = treasureCost;
     }
+
+    @Override
+    public String toString() {
+        return treasureTypes + ": " + String.format("%.2f", weight)
```

Comment:
Not sure but it could be that in latest java there are some utility functions for toString method. Please reserch